src/transport.ts

```
@@ -107,11 +111,11 @@ export function rainSolverTransport(
                         }).request(args);
                     } catch (error: any) {
                         if (shouldThrow(error)) throw error;
-                        if (tryNext) return req(false);
+                        if (tryNextCount) return req(--tryNextCount);
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)